app/main.py

```diff
@@ -1,34 +1,54 @@
+from __future__ import annotations
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int,
+                 is_alive: bool = True) -> None:
```

Comment:
when created, deck is always alive, doesn't it?
remove is_alive from parameters and just declare bellow self.is_alive = True